content/05.conclusions.md

```diff
@@ -1,6 +1,59 @@
+In addition to producing acceptable results, this divide-and-conquer approach has advantages, such as the ability to process long texts without strongly depending context length of the models.
+It was clear from our results, however, that depriving the tool from processing the context of the paragraph had some drawbacks.
+This was especially true in the Results and Methods sections, where the tool often added details that were present later in the text, or explained again an acronym that was previously defined.
+One way to address this issue would be to use chatbots, such as OpenAI's ChatGPT [@url:https://openai.com/blog/chatgpt/], that could process the entire manuscript using the same paragraph-by-paragraph approach while still considering the previously processed text.
+This would mimic the way a human reads a manuscript, inherit the benefits of processing one paragraph at a time, and potentially produce higher-quality revisions.
+
+
+In this preliminary work we focused on OpenAI's models mainly because of the simplicy in accessing the API.
+However, our tool could be adapted to others models, such as BLOOM [@arxiv:2211.05100], GLM [@arxiv:2210.02414], or Meta's OPT [@arxiv:2205.01068], which are open.
+Among all models offered by OpenAI, we found that only Davinci, the most advanced and capable model, was able to successfully revise a scholarly manuscript.
+These GPT-3 models offer a variety of parameters that can be used to control their behavior.
+For example, the models are stochastic, meaning that different revisions can be generated from the same input paragraph.
+This is a desirable property for our use case, as it allows the authors to generate multiple revisions of the same section and then choose the prefered one.
+Our tool, however, allows to easily control this randomness of the model by adjusting parameters such as the sampling temperature or use other techniques such as top-p (nucleus) sampling.
+Other parameters could be useful in improving the model outcomes, such as the `logit_bias` parameter, which allows to change the likelihood of specific tokens for a completion.
+An example would be to ban of the use of tokens related to references to figures or tables when we know these are not present in the paragraph to be revised.
+Another option in improving the revision outcomes would be to fine-tune the models using a specific corpus of text.
+This would allow to train even section-specific models that could be used to improve the revision of a particular section without the need to design a complex prompt (such as our prompt for the Methods section).
+
+
+This work lays the foundation for a future where academic manuscripts are constructed by a process that incorporates both human and machine authors.
+We believe that our AI-based revision tool, based on large language models, is a step towards a more efficient and effective way of writing scientific manuscripts.
```

Comment:
```suggestion
An AI-based revision tool, using large language models, is a step towards a more efficient and effective way of writing scientific manuscripts.
```
You haven't made the points on inclusion that I've seen yet. I think _that_ would be a good last paragraph. Opening up who can participate in the communication could be an AI-enabled win for equity.